content/en/blogs/kubernetes-fundamentals-part-1.md

```diff
@@ -0,0 +1,90 @@
+# Kubernetes Fundamentals Part one: A Visual Quick Start</h1>
+
+
+![68747470733a2f2f7170682e66732e71756f726163646e2e6e65742f6d61696e2d71696d672d3637363130313064303137363262633930353438363038616564353966313133](https://cdn.discordapp.com/attachments/854369708812140564/892077349087084614/unknown.png)
```

Comment:
Please store the images in this [folder](https://github.com/kubesphere/website/tree/master/static/images/blogs/en) instead of CDN.